contracts/TablelandTables.sol

```diff
@@ -146,24 +145,14 @@ contract TablelandTables is
         uint256 tableId,
         address controller
     ) external override whenNotPaused {
-        // table doesn't exist or
-        if (!_exists(tableId)) {
```

Comment:
I realized we don't actually need this check because the `ownerOf` call does it for us.